modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -463,7 +485,6 @@ struct CvCapture_FFMPEG
     bool setProperty(int, double);
     bool grabFrame();
     bool retrieveFrame(int, unsigned char** data, int* step, int* width, int* height, int* cn);
-    void rotateFrame(cv::Mat &mat) const;
```

Comment:
The logic related to frame rotation and orientation metadata should be restored.